mam/v2/prng/prng_types.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * MAM is based on an original implementation & specification by apmi.bsu.by
+ * [ITSec Lab]
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "mam/v2/prng/prng_types.h"
+#include "mam/v2/prng/mam_prng_t_set.h"
+
+size_t mam_prng_serialized_size() { return MAM2_PRNG_KEY_SIZE; }
+
+void mam_prng_serialize(mam_prng_t const *const prng, trits_t trits) {
+  trits_copy(trits_from_rep(MAM2_PRNG_KEY_SIZE, prng->secret_key), trits);
```

Comment:
I think we should use pb3 primitives for consistency

Reply:
sure